tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,229 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Icon,
+  Flex,
+  Button,
+  Section,
+  Collapsible,
+} from "../components";
+import { Window } from "../layouts";
+import { logger } from "../logging";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+  
+  return (
+    <Window
+      resizable
+      title={`Vote${
+        mode
+          ? `: ${
+            question
+              ? question.replace(/^\w/, c => c.toUpperCase())
+              : mode.replace(/^\w/, c => c.toUpperCase())
+          }`
+          : ""
+      }`}
```

Comment:
Eeeeh surely there's a way to make this read better right? Hell maybe this should even be a constant that does the replace and checks if it should be a question or a mode displaying beforehand too

Reply:
Smth like this would work much better
```js
let windowTitle = 'Vote';
if (mode) {
  windowTitle += ': ' + (question || mode).replace(/^\w/, c => c.toUpperCase());
}
// ...
<Window title={windowTitle}